### PR TITLE
[macOS] Software updates week 7 2021

### PR DESCRIPTION
**Pull request by dibir-magomedsaygitov**
# Description
- Xamarin.iOS - 14.10.0.4
- Xamarin.Mac - 7.4.0.4

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/macos/toolsets/toolset-10.15.json [hunk 1]
```diff
@@ -23,10 +23,10 @@
             "6.12.0.113", "6.10.0.106", "6.8.0.123", "6.6.0.166", "6.4.0.208"
         ],
         "ios-versions": [
-            "14.8.0.3", "14.6.0.15", "14.4.1.3", "14.2.0.12", "14.0.0.0", "13.20.2.2", "13.18.2.1", "13.16.0.13", "13.14.1.39", "13.10.0.21", "13.8.3.0", "13.6.0.12", "13.4.0.2", "13.2.0.47"
+            "14.10.0.4", "14.8.0.3", "14.6.0.15", "14.4.1.3", "14.2.0.12", "14.0.0.0", "13.20.2.2", "13.18.2.1", "13.16.0.13", "13.14.1.39", "13.10.0.21", "13.8.3.0", "13.6.0.12", "13.4.0.2", "13.2.0.47"
         ],
         "mac-versions": [
-            "7.2.0.3", "7.0.0.15", "6.22.1.26", "6.20.2.2", "6.18.3.2", "6.16.0.13", "6.14.1.39", "6.10.0.21", "6.8.3.0", "6.6.0.12", "6.4.0.2", "6.2.0.47"
+            "7.4.0.4", "7.2.0.3", "7.0.0.15", "6.22.1.26", "6.20.2.2", "6.18.3.2", "6.16.0.13", "6.14.1.39", "6.10.0.21", "6.8.3.0", "6.6.0.12", "6.4.0.2", "6.2.0.47"
```
Comment (maxim-lobanov) [inline images/macos/toolsets/toolset-10.15.json:29]: We also need to add new bundle below

Comment (dibir-magomedsaygitov) [reply images/macos/toolsets/toolset-10.15.json:29]: Thank you, fixed

File: images/macos/toolsets/toolset-10.15.json [hunk 2]
```diff
@@ -23,16 +23,23 @@
...
         ],
         "android-versions": [
             "11.1.0.26", "11.0.2.0", "10.3.1.4", "10.2.0.100", "10.1.3.7", "10.0.6.2"
         ],
-        "bundle-default": "6_12_5",
+        "bundle-default": "6_12_6",
```
Comment (maxim-lobanov) [inline images/macos/toolsets/toolset-10.15.json:34]: The new bundle requires Xcode 12.4 so I think we should keep `6_12_5` as default for now until Xcode is updated

Comment (dibir-magomedsaygitov) [reply images/macos/toolsets/toolset-10.15.json]: Thanks, changed to default bundle to `6_12_5`

**Review by miketimofeev — APPROVED** [review]

**Review by maxim-lobanov — APPROVED** [review]

**Review by MaksimZhukov — APPROVED** [review]

**Pull request merged**